Directory.Build.props

```diff
@@ -12,11 +12,12 @@
 		<CopyToLibraries Condition="'$(CopyToLibraries)'==''">true</CopyToLibraries>
 	</PropertyGroup>
 
-	<PropertyGroup Condition="'$(ResonitePath)'==''">
```

Comment:
The resonite stuff can in theory be moved to a Directory.Build.props in the root directory people use for their mod projects